src/core_plugins/kibana/public/context/index.js

```diff
@@ -0,0 +1,56 @@
+import uiRoutes from 'ui/routes';
+import './app';
+import { getDocumentUid } from './api/utils/ids';
+
+
+uiRoutes
+.when('/context/:indexPattern/:type/:id', {
+  controller: ContextAppRouteController,
+  controllerAs: 'contextAppRoute',
+  resolve: {
+    indexPattern: function ($route, courier, savedSearches) {
+      return courier.indexPatterns.get($route.current.params.indexPattern);
+    },
+  },
+  template: (
+    `<context-app
+      anchor-uid="contextAppRoute.anchorUid"
+      columns="contextAppRoute.state.columns"
+      discover-url="contextAppRoute.discoverUrl"
+      index-pattern="contextAppRoute.indexPattern"
+      predecessor-count="contextAppRoute.state.predecessorCount"
+      successor-count="contextAppRoute.state.successorCount"
+      sort="[contextAppRoute.indexPattern.timeFieldName, 'desc']"
+    >`
```

Comment:
Can we extract this into its own HTML file? This is more consistent with how we've been separating HTML and JS.